docs/gaia/gaia.rst

```diff
@@ -709,153 +660,213 @@ Now, you can query your table as follows:
+  >>> ["nobs","utype","utype sample"], \
+  >>> ["raj2000","flags","Ra"], \
+  >>> ["dej2000","flags","Dec"]])
   
-  Retrieving table 'user_<user_login_name>.my_table'
-  Parsing table 'user_<user_login_name>.my_table'...
+  Retrieving table 'user_joe.table'
+  Parsing table 'user_joe.table'...
   Done.
-  Table 'user_<user_login_name>.my_table' updated.  							
+  Table 'user_joe.table' updated.
 
+2.6. Cross match
+~~~~~~~~~~~~~~~~
 
-2.5. Tables sharing
-~~~~~~~~~~~~~~~~~~~
+It is possible to run a geometric cross-match between the RA/Dec coordinates of two tables
+using the crossmatch function provided by the archive. In order to do so the user must be
+logged in. This is required because the cross match operation will generate a join table
+in the user private area. That table contains the identifiers of both tables and the separation,
+in degrees, between RA/Dec coordinates of each source in the first table and its associated
+source in the second table. Later, the table can be used to obtain the actual data from both tables.
 
-You can share your tables to other users. You have to create a group, populate that group with users, and share your table to that group.
-Then, any user belonging to that group will be able to user your shared table in a query.
+In order to perform a cross match, both tables must have defined RA and Dec columns
+(Ra/Dec column flags must be set: see previous section to know how to assign those flags).
 
-2.5.1 Creating a group
-~~~~~~~~~~~~~~~~~~~~~~
+The following example uploads a table and then, the table is used in a cross match:
 
 .. code-block:: python
 
   >>> from astroquery.gaia import Gaia
   >>> Gaia.login()
-  >>> Gaia.share_group_create(group_name="my_group", description="description")
+  >>>
+  >>> table = file or astropy.table
+  >>> Gaia.upload_table(upload_resource=table, table_name='my_sources')
+  >>>
+  >>> # the table will be uploaded into the user private space into the database
+  >>> # the table can be referenced as <database user schema>.<table_name>
+  >>>
+  >>> full_qualified_table_name = 'user_<your_login_name>.my_sources'
+  >>> xmatch_table_name = 'xmatch_table'
+  >>> Gaia.cross_match(full_qualified_table_name_a=full_qualified_table_name, \
+  >>>               full_qualified_table_name_b='gaiadr2.gaia_source', \
+  >>>               results_table_name=xmatch_table_name, radius=1.0)
+  >>>
 
-2.5.2 Removing a group
-~~~~~~~~~~~~~~~~~~~~~~
+Once you have your cross match finished, you can obtain the results:
+
+.. code-block:: python
+
+  >>> xmatch_table = 'user_<your_login_name>.' + xmatch_table_name
+  >>> query = 'SELECT c."dist"*3600 as dist, a.*, b.* FROM gaiadr2.gaia_source AS a, '+\
+  >>> full_qualified_table_name+' AS b, '+\
+  >>> xmatch_table+' AS c '+\
+  >>> 'WHERE (c.gaia_source_source_id = a.source_id AND c.my_sources_my_sources_oid = b.my_sources_oid)'
+  >>> job = Gaia.launch_job(query=query)
+  >>> results = job.get_results()
```

Comment:
this won't work, please change to fstrings or string formatting, and use `...` for starting the multilines